### PR TITLE
Modernized the examples a bit..

### PR DESCRIPTION
**Pull request by marcoscaceres**
Hopefully using ES6 doesn't break ReSpec's syntax highlighting.

### CODE REVIEW

File: geofencing-spec.html [hunk 1]
```diff
@@ -93,32 +93,29 @@
...
-        // would be the time to store this in some storage.
-      }, function(error) {
-        // During development it often helps to log errors to the
-        // console. In a production environment it might make sense to
-        // also report information about errors back to the
-        // application server.
-        console.log(error);
-      }
-    );
+'use strict';
```
Comment (mkruisselbrink) [inline geofencing-spec.html]: Any particular reason to have 'use strict' here (but not in any of the other examples?)

Comment (marcoscaceres) [inline geofencing-spec.html]: No. I can remove it.

File: geofencing-spec.html [hunk 2]
```diff
@@ -93,32 +93,29 @@
...
-    serviceWorkerRegistration.geofencing.add(
-        new CircularGeofenceRegion({
-          name: "myfence",
-          latitude: 37.421999,
-          longitude: -122.084015,
-          radius: 1000
-        }), {includePosition: true}).then(
-      function(geofence) {
-        console.log(geofence.id);
-        // If more than just a name needs to be stored with a geofence, now
```
Comment (mkruisselbrink) [inline geofencing-spec.html]: I think it would be good if somehow this comment could be kept, as it kind of explains what the ID is good for

**Comment by marcoscaceres** [comment]
Re-included the comments. I didn't re-include the error comments as I don't think they are needed.

**Comment by marcoscaceres** [comment]
Thanks for including my changes :)

**Pull request merged**